dask_cuda/initialize.py

```diff
@@ -18,11 +18,11 @@
 def _create_cuda_context_handler():
     if int(os.environ.get("DASK_CUDA_TEST_SINGLE_GPU", "0")) != 0:
         try:
-            numba.cuda.current_context()
-        except numba.cuda.cudadrv.error.CudaSupportError:
+            cuda.core.experimental.Device().set_current()
```

Comment:
This appears to be equivalent to [`numba.cuda.current_context()`](https://github.com/NVIDIA/numba-cuda/blob/c48e2130cbbc9d4f903cac5f07a03580a5146074/numba_cuda/numba/cuda/api.py#L20), which uses the cuda runtime to get or create a context: https://github.com/NVIDIA/numba-cuda/blob/c48e2130cbbc9d4f903cac5f07a03580a5146074/numba_cuda/numba/cuda/cudadrv/devices.py.